jolly-jellyfish/src/page_maker/templates/page_maker/webpage_view.html

```diff
@@ -5,18 +5,31 @@
 {% endblock %}
 
 {% block content %}
-<div><h1>{{ webpage.user_title }}</h1></div>
-<img src="{{ webpage.user_image_1.url }}">
-<div>{{ webpage.user_text_1 }}</div>
-<img src="{{ webpage.user_image_2.url }}">
-<div>{{ webpage.user_text_2 }}</div>
-<img src="{{ webpage.user_image_3.url }}">
-<div>{{ webpage.user_text_3 }}</div>
-<img src="{{ webpage.user_image_4.url }}">
+<h1><span>{{ webpage.user_title }}</span></h1>
```

Comment:
```suggestion
<h1><span><span>{{ webpage.user_title }}</span></span></h1>
```
Missing a level of spans